### ISSUE TITLE
Support Arrays of types

### ISSUE THREAD

**Issue by wesbos** [tamj0rd2/ts-quickfixes#76]
First off, thanks a million for the extension! super handy.

I've found an array of type doesn't work. Take this code:

```ts
interface Job {
  title: string;
  location: string;
}

interface Person {
  name: string;
  age: number;
  job: Job[];
}

const wes: Person = {};
```

If I change Person `Job[]` to `Job`, it does do the nested values. 

Could it work for arrays of Job as well?

**Comment by tamj0rd2** [comment]
Hey, thanks for the feedback. If you trigger the quick fix on your `wes` variable, this should be the result:

```typescript
const wes: Person = {
    name: 'todo',
    age: 0,
    job: []
};
```

I'm not sure if it should automatically create a `Job` object inside of that array. Some people might not expect that kind of behaviour. Maybe this would be a better solution?

1. You use the declare missing members fix to fill out the missing members, same as above
2. You write an empty object inside of the job array, like this:

```typescript
const wes: Person = {
    name: 'todo',
    age: 0,
    job: [{}]
};
```

3. While your cursor is inside of that empty object, you trigger the declare missing members fix again

Does that sound like a decent compromise? It doesn't work right now, but it's something I'd be happy to add when I have time, or accept as a PR contribution.

**Comment by wesbos** [comment]
Oh! I was trying to trigger it inside the object. It does work like you have said.

I'm not seeing how to trigger it inside to create the `Job` missing members though, does my cursor need to be on the `job` part of `job: []` ?

I think option 3 would be what I expect. I don't think that would create an unexpected results, but I haven't totally thought it trough

**Comment by tamj0rd2** [comment]
Ok cool! I don't think I explained it so well. The 3 points I wrote above were supposed to be the steps you'd ideally take to do what you're asking for, rather than 3 different solutions.

> I'm not seeing how to trigger it inside to create the Job missing members though, does my cursor need to be on the job part of job: [] ?

```typescript
const wes: Person = {
    name: 'todo',
    age: 0,
    job: [{}]
};
```

At this stage you would put your cursor inside of `{}` to trigger the quick fix, but it doesn't work at the moment. There needs to be a code change first, but it's something I'm happy to add
